Node.js/GettingStarted/processcsv.py

```diff
@@ -5,96 +5,77 @@
 import os
 import argparse
 
-#### TODO: Get the configuration
+#### Recommendation: Store this as environment variables or download it from key vault
 
 storage_acc_name = '<storage account name>'
 storage_acc_key = '<storage account key>'
 block_blob_service = BlockBlobService(account_name=storage_acc_name,account_key=storage_acc_key)
 ####
 
 def getfilename(name):
-
     names = str(name).split('/')
     return names[len(names)-1]
 
-
-
 def processcsvfile(fname,seperator,outdir,outfname):    
-    header = []    
-    with open(fname) as inpfile:
-        
+    header = []
+    isHeaderLoaded = False    
+    with open(fname) as inpfile:        
         allrows = csv.reader(inpfile,delimiter=seperator)
         print("loaded file " + fname)
         all_vals = []
-
         for rows in allrows:
```

Comment:
I am pretty sure this is a distinct unit of functionality - consider pulling it out into a meaningfully named function.